RetrospectiveExtension.Frontend/components/__tests__/mocked_components/WorkItemTracking.tsx

```diff
@@ -0,0 +1,81 @@
+import { mocked } from 'jest-mock';
```

Comment:
I'm open to better names for this folder than `mocked_components` ... essentially these components are mocked implementations of classes within the WorkItemTracking class in the auzre api. However since they are not functions / services, we can use these implementations in-line in tests without needing `jest.mock`